tests/xharness/Jenkins.cs

```diff
@@ -2345,7 +2345,7 @@ public void CloneTestProject (TestProject project)
 
 		public string LogDirectory {
 			get {
-				var rv = Path.Combine (Jenkins.LogDirectory, $"{TestName}_{ID}");
+				var rv = Path.Combine (Jenkins.LogDirectory, $"{TestName}", $"{ID}");
```

Comment:
Idk if the compiler is smart enough to do it (or not) but `${TestName}` and `${ID}` are potentially more costly than using `TestName` and `ID` directly.